frontend/src/views/cp_control_panel/ControlPanel.jsx

```diff
@@ -0,0 +1,22 @@
+import React from "react";
+import { PositionsList } from "../../components/positions-list";
+import { FilteredList } from "../../components/filtered-list";
+import { connect } from "react-redux";
+import { positionsSelector } from "../../api/actions";
+import { ConnectedSessionSelect } from "../sessions/ConnectedSessionSelector";
+
+function ControlPanel() {
+    const ConnectedFilteredList = connect((state, ownProps) => ({
```

Comment:
Connect outside this function. Otherwise, every time the component re-renders it will be re`connect`ed...